src/components/Sidebar/Sidebar.js

```diff
@@ -1,24 +1,65 @@
-import React from "react";
-import { LogoWrapper, StyledSidebar } from "./Sidebar.styled";
-import { MdEventNote, MdAdd } from "react-icons/md";
+import { useEffect, useState } from "react";
+
+import { NavLink } from "react-router-dom";
+
+import { Button, Divider, Drawer, Tab, Tabs, Typography } from "@mui/material";
+import CalendarTodayIcon from "@mui/icons-material/CalendarToday";
+import AddRoundedIcon from "@mui/icons-material/AddRounded";
+
+const drawerWidth = 200;
 
 const Sidebar = () => {
+  const [activeValue, setActiveValue] = useState(0);
+
+  useEffect(() => {
+    const pathName = window.location.pathname;
+    if (pathName === "/" && activeValue !== 0) {
+      setActiveValue(0);
+    } else if (pathName === "/add" && activeValue !== 1) {
+      setActiveValue(1);
+    }
+  }, [activeValue]);
+
+  const activeValueHandler = (e, value) => {
+    setActiveValue(value);
+  };
+
   return (
-    <StyledSidebar>
-      <LogoWrapper>LOGO</LogoWrapper>
-      <nav>
-        <ul>
-          <li className="active">
-            <MdEventNote size={30} />
-            <p>Events</p>
-          </li>
-          <li>
-            <MdAdd size={30} />
-            <p>Add</p>
-          </li>
-        </ul>
-      </nav>
-    </StyledSidebar>
+    <Drawer
+      variant="permanent"
+      sx={{
+        width: drawerWidth,
+        flexShrink: 0,
+        [`& .MuiDrawer-paper`]: { width: drawerWidth, boxSizing: "border-box" },
```

Comment:
please add this to the global theme. Avoid inline styles in components

Reply:
Still searching for a method to override Material UI default styling.